lib/declare_schema.rb

```diff
@@ -21,6 +22,8 @@ class Boolean; end
     text:     String
   }.freeze
 
+  SEMVER_8 = Gem::Version.new('8.0.0').freeze
```

Comment:
I'm cheating slightly to use `Gem::Version`. I've found it really handy for SemVers in general, since it implements comparison with any number of decimals.